### DISCUSSION TITLE
Github Down

### DISCUSSION THREAD

**Discussion by david-engelmann** [community/community#60726] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Github is no longer accepting commits from local and is returning the following error after a long failed attempt

```
ssh: Could not resolve hostname github.com: Temporary failure in name resolution
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
I've confirmed that access isn't the issue. Momentarily, on the desktop version, I'm seeing a red alert bar at the top that says essentially, can't guarantee all commits are present

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by david-engelmann** [comment, marked as answer]
Restart computer :)

**Reply by Sam04tt** [reply]
[Public roadmap] 0x574b3584DebD8d50034bB9d01f2db20a0edB0570
Mobile
Question: 0x574b3584DebD8d50034bB9d01f2db20a0edB0570

**Comment by iamKingnoob** [comment]
hi  poo
